aspnet/identity/overview/features-api/customize_identity_model.md

```diff
@@ -298,27 +310,27 @@ public class ApplicationDbContext : IdentityDbContext<ApplicationUser>
 }
 ```
 
-And update Startup.cs to use the new ApplicationUser class:
+Update `ConfigureServices` to use the new `ApplicationUser` class:
 
 ```CSharp
 services.AddDefaultIdentity<ApplicationUser>()
     .AddEntityFrameworkStores<ApplicationDbContext>();
 ```
 
-There is no need to override OnModelCreating here because EF Core will map the `CustomTag` property by convention. However, the database will need to be updated to get a new CustomTag column. To do this, add a migration, and then update the database as described at the top of this article.
+There is no need to override `OnModelCreating` here because EF Core will map the `CustomTag` property by convention. However, the database will need to be updated to use the new `CustomTag` column. For more information, see [Identity and EF Core Migrations](#identity-migrations).
```

Comment:
Don't really understand this edit. What does it mean for a "database to use a column"?